src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QuerySettingsHelper.cs

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.IO;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+using Microsoft.SqlTools.Hosting.Protocol;
+using Microsoft.SqlTools.ServiceLayer.Connection;
+using Microsoft.SqlTools.ServiceLayer.Connection.Contracts;
+using Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts;
+using Microsoft.SqlTools.ServiceLayer.QueryExecution.Contracts.ExecuteRequests;
+using Microsoft.SqlTools.ServiceLayer.QueryExecution.DataStorage;
+using Microsoft.SqlTools.ServiceLayer.SqlContext;
+using Microsoft.SqlTools.ServiceLayer.Workspace;
+using Microsoft.SqlTools.ServiceLayer.Workspace.Contracts;
+using Microsoft.SqlTools.ServiceLayer.Hosting;
+using Microsoft.SqlTools.Utility;
+using System.Diagnostics;
+using System.Xml.Serialization;
```

Comment:
Do you really need all these?

Reply:
Done.  Copy\paste issue.  Unfortunately VS Code doesn't have "Remove unneeded using" like VS, so you have to manually figure out which ones you need.

Reply:
I poked around a bit and turns out there is - it's just hidden under a random keybinding. ctrl + . will give you that option :

![image](https://user-images.githubusercontent.com/28519865/57890645-32845a80-77ed-11e9-8280-999dafc29a2a.png)